### PR TITLE
Accept

### PR DESCRIPTION
**Pull request by johnbradley**
Changes for accept UI.

### CODE REVIEW

**Comment by johnbradley** [comment]
@dleehr can you review this pull request.
Thanks.

File: handover_api/utils.py [hunk 1]
```diff
@@ -31,9 +32,10 @@ def send_draft(draft):
     message = generate_message(sender.email, receiver.email, subject, template_name, context)
     message.send()
 
+
 def get_accept_url(handover):
-    # TODO: lookup the accept url
-    return 'https://itlab-1.gcb.duke.edu/accept?token=' + str(handover.token)
+    return reverse('accept-process') + "?token=" + str(handover.token)
```
Comment (dleehr) [inline handover_api/utils.py]: I think reverse just returns a relative URL - we need the protocol and host too.

File: switchboard/dds_util.py [hunk 2]
```diff
@@ -1,8 +1,10 @@
 from ddsc.config import Config
 from django.conf import settings
 from ddsc.core.remotestore import RemoteStore
+from ddsc.core.util import KindType
```
Comment (dleehr) [inline switchboard/dds_util.py]: I don't see `KindType` used in the code

**Comment by dleehr** [comment]
Nice work!

File: handover_api/utils.py (hunk 1 above)
Comment (dleehr) [inline handover_api/utils.py]: Determining the absolute URL is probably outside the scope of this feature. Within the framework, we'd either need a request object or to manually configure the sites framework. We can just put the base URL into settings.py

For now, just prefix the relative path with the hard-coded base URL.

**Comment by dleehr** [comment]
This implements functionality for #7

**Pull request merged**